lib/snapshot/BUILD.gn

```diff
@@ -185,16 +185,15 @@ if (host_os == "mac" && target_os != "mac" &&
 # platform.
 #
 # This target is used for builds targeting macOS.
```

Comment:
Outdated comment.

Reply:
Fixed -- thanks! Once we flip the switch for Android, I'll update to artifacts for macOS hosts.